packages/truffle-core/lib/commands/test.js

```diff
@@ -38,6 +38,10 @@ const command = {
       {
         option: "--show-events",
         description: "Log all contract events."
+      },
+      {
+        option: "--quiet",
+        description: "Suppress all compilation output."
```

Comment:
Hmm, should this suppress the migration report as well? Seems kind of weird for this option to only pertain to compilation.

Reply:
Hmmm, good point. 🤔